examples/winit.rs

```diff
@@ -1,10 +1,30 @@
 use std::num::NonZeroU32;
+
 use winit::event::{Event, WindowEvent};
-use winit::event_loop::{ControlFlow, EventLoop};
+use winit::event_loop::{ControlFlow, EventLoop, EventLoopBuilder};
+#[cfg(target_os = "android")]
+pub use winit::platform::android::{activity::AndroidApp, EventLoopBuilderExtAndroid};
 use winit::window::WindowBuilder;
 
+/// Run with `cargo apk r --example winit_android`
```

Comment:
We have a mention of how to run the example on webassembly in the readme: https://github.com/rust-windowing/softbuffer/#webassembly.

So a quick mention of Android there would be good.